docker/apache.conf

```diff
@@ -0,0 +1,14 @@
+# Configure apache
+<VirtualHost *:80>
+    ServerName example.com
```

Comment:
Hey, it shouldn't need to be changed, but actually thinking about it, it would be better if the ServerName line didn't exist at all. Since this is the only site in the container, and not a shared hosting container, it doesn't apply at all.